en/drafts/originals/designing-a-timeline-tabletop-simulator.md

```diff
@@ -64,354 +60,352 @@ In 2013, game designer Eric Zimmerman published a “Manifesto for a Ludic Centu
 
 In this lesson, you will learn how to make you own version of a tabletop card game called _Timeline_.
 
 ### What is _Timeline_?
 
-_Timeline_ is a card game designed by Frédéric Henry, first published in 2012 and still available for purchase through the global game publisher, [Asmodee](https://www.asmodee.ca/product/timeline-access/) and its subsidiary [Zygomatic](https://www.dobblegame.com/en/games/#category-timeline). _Timeline_ can be played with 2 to 8 players. As it is a very simple game to learn to play with a group, _Timeline_ is frequently described as a party game as much as an educational game. The game is published in several languages and there are a number of differently themed versions of the game that have been published over the last ten years including, _Timeline: Inventions_, _Timeline: Music and Cinema_, _Timeline: American History_, and _Timeline Star Wars_.
+_Timeline_ is a card game designed by Frédéric Henry, first published in 2012 and still available for purchase through the global game publisher [Asmodee](https://www.asmodee.ca/product/timeline-access/) and its subsidiary [Zygomatic](https://www.dobblegame.com/en/games/#category-timeline). _Timeline_ can be played with 2 to 8 players. As it is a very simple game to learn to play with a group, _Timeline_ is frequently described as a party game as much as an educational game. The game is published in several languages and in a number of differently themed versions, including _Timeline: Inventions_, _Timeline: Music and Cinema_, _Timeline: American History_, and _Timeline Star Wars_.
 
-> **_Timeline_** is a card game where each card, on both sides, depicts a specific historical event, invention or discovery, with only one side displaying the year in which that event occurred. Players take turns placing a card from their hand in a row on the table. After placing the card, the player reveals the date on it. If the card was placed correctly with the date in chronological order with all other cards on the table, the card stays in place; otherwise the card is removed from play and the player takes another card from the deck. The first player to get rid of all their cards by placing them correctly wins.
+> **_Timeline_** is a card game where each card depicts a specific historical event, invention or discovery, but only one side displays the year in which that event occurred. Players take turns placing a card from their hand in a row on the table. After placing the card, the player flips it to reveal the date on the back. If the card was placed in chronological order with all the other cards on the table, it stays in place; otherwise the card is removed from play and the player takes another card from the deck. The first player to get rid of all their cards by placing them correctly wins.
```

Comment:
I'm not certain whether this passage is a citation or not. Some of the phrasing is similar to a description of _Timeline_ gameplay I found on [BoardGameGeek](https://boardgamegeek.com/boardgame/128664/timeline). Is this where you found it originally?

Reply:
I checked and found that I have previously bookmarked the same link to BoardGameGeek, so while I cannot recall if I got the phrasing from the site, I'd rather err on the side of caution and add a citation if the language is close. Thank you!